site/pages/our-work/case-study/nandos-loyalty/index.js

```diff
@@ -0,0 +1,379 @@
+// @flow
+import React from 'react';
+
+// import Social from '../../../../components/social';
+import Image from '../../../../components/image';
+import ScrollTracker from '../../../../components/scroll-tracker';
+// import Triptych from '../../../../components/triptych';
+import {
+  PageHeading,
+  Body,
+  Quote,
+  Section,
+  SectionHeading,
+  SectionBody,
+  P,
+  H3,
+  UL,
+  // ArticleImg,
+} from '../shared';
+// import Video from '../shared/video';
+import WhatToReadNext from '../shared/what-to-read-next';
+import ChecklistContactUs from '../../../../slices/checklist-contact-us-slice';
+import ListBox from '../../../../components/list-box';
+import Card from '../../../../components/card';
+
+// Image imports
+import DevicesX1Img from './images/Devices@1x.png';
+import DevicesX2Img from './images/Devices@2x.png';
+import HeaderX1Img from './images/Header@1x.png';
+import HeaderX2Img from './images/Header@2x.png';
+import HeaderX3Img from './images/Header@3x.png';
+import PlasticCardX1Img from './images/PlasticCard@1x.png';
+import PlasticCardX2Img from './images/PlasticCard@2x.png';
+import RyanX2Img from './images/Ryan@2x.png';
+import TapWithPhoneX1Img from './images/TapWithPhone@1x.png';
+import TapWithPhoneX2Img from './images/TapWithPhone@2x.png';
+import TapWithWatchX1Img from './images/TapWithWatch@1x.png';
+import TapWithWatchX2Img from './images/TapWithWatch@2x.png';
+import TestingX1Img from './images/Testing@1x.png';
+import TestingX2Img from './images/Testing@2x.png';
+
+import styles from './style.css';
+// import Triptych from './triptych';
```

Comment:
I was aiming to re-enable this code for use later, we should get rid of it if we're not using it.